openquakeplatform/openquakeplatform/common/views.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from django.shortcuts import render_to_response
+from django.template import RequestContext
+from importlib import import_module
+
+from openquakeplatform.settings import INSTALLED_APPS
+
+
+def versions(request, **kwargs):
+    page_title = 'Application Versions'
+
+    apps_list = []
+    for app_name in INSTALLED_APPS:
+        if not app_name.startswith('openquakeplatform'):
+            continue
+
+        app = import_module(app_name)
+        header_info = getattr(app, 'header_info', None)
+        vers = getattr(app, '__version__', None)
+
+        if header_info:
+            name = header_info['title']
+        else:
+            name = app_name
+
+        if vers is None:
```

Comment:
Perhaps this check should be moved before the previous one? If you don't do the append, you don't need to get the name.